pydocteur/github_api.py

```diff
@@ -9,7 +9,7 @@
 from pydocteur.settings import REPOSITORY_NAME
 
 logger = logging.getLogger("pydocteur")
-gh = Github(GH_TOKEN)
+gh = Github(GH_TOKEN if GH_TOKEN else None)
```

Comment:
Not needed, if GH_TOKEN not here then the code won't run. see https://github.com/AFPy/PyDocTeur/blob/main/pydocteur/settings.py#L35-L39

Reply:
The tests pass without a token, and I don't want a token in the tests. Should we word this in some way to make it explicit?

Reply:
Just tested and this is what i got: https://wyz.fr/4F-1M

From adding https://wyz.fr/0I-NN

Reply:
Because `get_rest_api` uses `HTTPBasicAuth` to bypass rate limiting

Reply:
(This is tested on the `main` branch, not on yours BTW)

Reply:
I don't follow, how we would care about rate limiting in the tests? They don't hit the actal API (they the cassettes).